runtime/src/enclave_rpc/types.rs

```diff
@@ -25,7 +25,6 @@ impl SessionID {
 /// Frame.
 #[derive(Debug, Serialize, Deserialize)]
 pub struct Frame {
-    #[serde(with = "serde_bytes")]
```

Comment:
Note: `serde_bytes` now internally uses `serde_bytes::{Serialize,Deserialize}` traits, and is missing implementations for fixed sized byte arrays, so we cannot use it here anymore. 

However `serde_bytes` is not even needed here since we implement "byte-aware" serde serialization and deserialization methods in `impl_bytes!` anyway.